packages/engine/lib/stateful/src/field/key.rs

```diff
@@ -31,7 +36,7 @@ impl RootFieldKey {
     ///
     /// # Errors
     ///
-    /// - Returns [`Error`] if name starts with [`PRIVATE_PREFIX`] or [`HIDDEN_PREFIX`]
+    /// - Returns [`Error`] if `name` starts with a prefixed pre-defined by [`FieldScope`].
```

Comment:
I can't seem to suggest changes, but prefix* not prefixed

Reply:
Applied in 0f1b85bc36d0a36fcf5a5ca7cb8ecf0fd5bc90b6